advanced/folder-sync-xattrs.rst

```diff
@@ -0,0 +1,38 @@
+syncXattrs
+==========
+
+.. versionadded:: 1.22.0
+
+When enabled, Syncthing attempts to also synchronise filesystem extended
```

Comment:
```suggestion
When enabled, Syncthing attempts to also synchronize filesystem extended
```

(not completely consistent in the docs, but the American spelling has more prior occurrences)